.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,30 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches:
+      - master
+      
+jobs:
+  cancelPreviousRuns:
+    name: Cancel previous runs
+    if: github.event_name == 'push' || !github.event.pull_request.draft
+    runs-on: macos-latest
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.6.0
```

Comment:
should this be better: @0.8.0? In the template for Android was 0.6.0, but in the official website is already 0.8.0 https://github.com/styfle/cancel-workflow-action

Reply:
Yes, we can always update to the latest version. We can also tell Dependabot to create PRs to update them automatically: https://grandcentrix.atlassian.net/wiki/spaces/TENG/pages/31183710590/GitHub+Dependabot

Reply:
No update? 🙂